internal/cli/atlas/deployments/setup.go

```diff
@@ -730,10 +734,56 @@ func (opts *SetupOpts) Run(ctx context.Context) error {
 	return opts.runConnectWith(cs)
 }
 
+func (opts *SetupOpts) RunAtlas(ctx context.Context) error {
+	s := setup.Builder()
+
+	// remove global flags and unknown flags
+	var newArgs []string
+	_, _ = log.Debugf("Removing flags and args from original args %s\n", os.Args)
+	newArgs, err := workflows.RemoveFlagsAndArgs(map[string]string{flag.TypeFlag: "1"}, map[string]bool{opts.DeploymentName: true}, os.Args)
+	if err != nil {
+		return err
+	}
+
+	// replace deployment name with cluster name
+	if opts.DeploymentName != "" {
+		newArgs = append(newArgs, fmt.Sprintf("--%s", flag.ClusterName))
+		newArgs = append(newArgs, opts.DeploymentName)
+	}
+
+	// remove atlas deployments from args
+	newArgs = newArgs[2:]
+	// update args
+	s.SetArgs(newArgs)
+
+	// run atlas setup
+	_, _ = log.Debugf("Starting to run atlas setup with args %s\n", newArgs)
+	_, err = s.ExecuteContextC(ctx)
+	return err
+}
+
+func (opts *SetupOpts) Run(ctx context.Context) error {
+	if err := opts.validateAndPrompt(); err != nil {
+		if errors.Is(err, errSkip) {
```

Comment:
where is errSkip defined?  I have been using `options.ErrDeploymentNotFound` so far. Wondering if I should use `errSkip` instead

Reply:
errSkip                         = errors.New("setup skipped")
we use this when customer picks the skip option

Reply:
Should we also check for `options.ErrDeploymentNotFound`. When using the indexes command the validateAndPrompt would fail with that error when running the command with Atlas